gulp-core-build/src/jsonUtilities/SchemaValidator.ts

```diff
@@ -66,6 +58,14 @@ export class SchemaValidator {
     return rawConfig as TResult;
   }
 
+  private static getFormattedErrorMessage(errors: Validator.SchemaErrorDetail[], dataFilePath?: string): string {
```

Comment:
Because this is static, can you just put it in a different module that isn't exported so you don't have to do the nasty `as any` below?